### DISCUSSION TITLE
Alert notification when resource is suspended / resumed

### DISCUSSION THREAD

**Discussion by havard024** [fluxcd/flux2#3144] [Proposals] [answered]
Using flux suspend, you can stop update of any resource in the cluster, that's great for testing, but if someone does this in production, there is no way of knowing why deployments are not happening unless you check for suspended resources.

It would be nice if it was possible to receive alerts / notifications if a resource is suspended or has been suspended for more than x amount of time.

That way, you will get a constant reminder that you have suspended resources, so you don't forget to resume them

**Comment by stefanprodan** [comment, marked as answer]
> It would be nice if it was possible to receive alerts / notifications if a resource is suspended or has been suspended for more than x amount of time.

You can use the `gotk_suspend_status` metric and setup a Prometheus alert using `for: 30m` or whatever interval suits you. 

Docs: https://fluxcd.io/flux/guides/monitoring/#metrics

**Reply by havard024** [reply]
Thanks for the reply, that will solve what I am after for sure, sorry I missed that from the documentation.
